bil.ott

```diff
@@ -714,3 +712,15 @@ defns reduce_stmt :: '' ::=
 
  ------------------------------------------------------------- :: seq_nil
  delta,word |- {} ~> delta, word, {}
+ 
+defns multistep_exp :: '' ::=
+
+ defn delta |- exp ~>* exp' :: :: mexp :: '' by
+
+ ---------------- :: refl
+ delta |- e ~>*  e
+
+ delta1 |- e1 ~> e2
+ delta2 |- e2 ~>* e3
```

Comment:
why are we using different deltas here? and why `~>*` occurs in the premises? To my understanding it should be `reduce (x,y) -> reduce (y,z) -> reduce (x,z)`.

Reply:
You are correct - sorry for this error (it's a hold-over from the previous iteration that had the expression reduction relation modify the environment).

I'll fix this and add a sentence or two explaining the ~>* relation to the latex file.